src/composables/secureStorageRef.ts

```diff
@@ -20,29 +22,38 @@ export function useSecureStorageRef<T = string | object | any[]>(
     'serializer' | 'enableSecureStorage' | 'onBackgroundSync'
   > = {},
 ) {
-  async function getDecryptedValue(encryptedValue: any) {
-    if (IS_MOBILE_APP || !encryptedValue) {
-      return encryptedValue;
+  /**
+   * Checks if the value needs to be decrypted and decrypts it using the encryption key.
+   * On mobile, the value is already decrypted by the storageRef composable.
```

Comment:
To be honest I would consider to encrypt the mnemonic also on the mobile. As far as I know from a research I did some time ago the biometric login provides a `key` that can be used.

My idea on the authentication architecture was to have two separate modules:
1. Encoding/decoding key provider - interface that takes any string and uses it to encode/decode data. As a source of the key we could use any source, like a password or biometric login token.
2. Encoder/decoder - get's the key generated by provider and allows to operate on any data.

Maybe we can create a separate task to research this in the future? Or at least add `TODO` comment here?

Reply:
We're encrypting the mnemonic on mobile through the `@aparajita/capacitor-secure-storage` package in the `SecureMobileStorage.ts` lib. Our used authentication package does not provide us a `token`.

Reply:
OK, ignore my comment. I'll create a task for the future.

My idea was not to use any external package for mobile storage but use the same logic on every env. This way the only matter is what we use as the "password" (the key). On web/extension this would be user's password and on the mobile the token received by the **biometric** login (not the secure storage). So when user uses the fingerpirint/face scanner the lib passes the token created at the first time we allowed the app to use biometric login. So it's like a password :) At least this was my assumption.